doc/classes/TextEdit.xml

```diff
@@ -101,6 +103,8 @@
 			<argument index="3" name="wrap_index" type="int" default="0">
 			</argument>
 			<description>
+				Moves the cursor at the specified line index.
+				If [code]adjust_viewport[/code] is set to true, the viewport will center at the cursor position after the move occurs.
```

Comment:
Same as above: 
"line" should be formatted as "adjust_viewport" below: surround with [code][/code].
Also: 2 parameters descriptions are missing: can_be_hidden and wrap_index

Reply:
I'm not sure what wrap_index is.

Reply:
I guess this is the column number where the TextEdit will wrap to a new line. That's useful for source code if you don't want a line of code to have length bigger than `wrap_index`.
**This is just a guess, it probably requires a test first.**